main.py

```diff
@@ -157,13 +178,29 @@ def main():
         help="Performs tangent operation",
         action="store_true")
     group.add_argument(
+        "-fbin", "--fbin",
+        help="Convert to binary",
+        action="store_true")
+    group.add_argument(
+        "-fhex", "--fhex",
```

Comment:
It doesn't make sense to pass the same variable for both `-` and `--`.
You can do `-fh` and `--fhex`. Same for the rest?